### PR TITLE
sessionToken oie limitations add

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** adding sessionToken limitation during upgrade
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-443007](https://oktainc.atlassian.net/browse/OKTA-443007)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md [hunk 1]
```diff
@@ -98,6 +98,14 @@ In Classic Engine, when a user is using both the Forgot Password Question and a
 
 ***
 
+#### Session Token created during an Identity Engine upgrade prompts user for password after upgrade completes
+
+**What Changed:** If a user authenticates in the Classic Engine (creating a `sessionToken`), and the upgrade to the Identity Engine completes during the time that the `sessionToken` is valid (five minutes), then when a user attempts to access an OpenID Connect app after the upgrade, the user is prompted to authenticate again.
```
Comment (shayekhazaeli-okta) [inline packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md:103]: The user is prompted for password again

Comment (roshanrajan-okta) [reply packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md:103]: +1

File: packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md [hunk 2]
```diff
@@ -61,19 +61,19 @@ The following Event Types are available only in the Identity Engine and can't be
...
 
-In Classic Engine, when a user is using both the Forgot Password Question and a Security Question for MFA, and an API call is made to `v1/lifecycle/reset_factors` to reset all the factors for the user, only the Security Question is reset. And then, if the GET `/factors` API is called, the Forgot Password Question isn't returned as a factor. With an upgrade to Identity Engine, after resetting all the factors, when the GET `/factors` API is then called, the Forgot Password Question is returned as a factor in the response.
+In Classic Engine, when a user is using both the Forgot Password Question and a Security Question for MFA, and an API call is made to `v1/lifecycle/reset_factors` to reset all the factors for the user, only the Security Question is reset. And then, if the GET `/factors` API is called, the Forgot Password Question isn't returned as a factor. With an upgrade to the Identity Engine, after resetting all the factors, when the GET `/factors` API is then called, the Forgot Password Question is returned as a factor in the response.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md:74]:
```suggestion
In Classic Engine, when a user uses both the Forgot Password Question and a Security Question for MFA, and an API call is made to `v1/lifecycle/reset_factors` to reset all the factors for the user, only the Security Question is reset. And then, if the GET `/factors` API is called, the Forgot Password Question isn't returned as a factor. With an upgrade to the Identity Engine, after resetting all the factors, when the GET `/factors` API is then called, the Forgot Password Question is returned as a factor in the response.
```

File: packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md [hunk 3]
```diff
@@ -98,6 +98,14 @@ In Classic Engine, when a user is using both the Forgot Password Question and a
 
 ***
 
+#### Session Token created during an Identity Engine upgrade prompts user for password after upgrade completes
```
Comment (jadsiblini-okta) [inline packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md:101]: nit: I'd suggest to change during by before

File: packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md [hunk 4]
```diff
@@ -98,6 +98,14 @@ In Classic Engine, when a user is using both the Forgot Password Question and a
...
+
+**What Changed:** If a user authenticates in the Classic Engine (creating a `sessionToken`), and the upgrade to the Identity Engine completes during the time that the `sessionToken` is valid (five minutes), then when a user attempts to access an OpenID Connect app after the upgrade, the user is prompted for their password again.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md:103]:
```suggestion
**What Changed:** If a user authenticates in the Classic Engine (by creating a `sessionToken`), and the upgrade to the Identity Engine completes during the time that the `sessionToken` is valid (five minutes), then when a user attempts to access an OpenID Connect app after the upgrade, the user is prompted for their password again.
```

**Review by christinechoy-okta — APPROVED** [review]
Just a nit...otherwise, LGTM! 🚢

**Pull request merged**